index.js

```diff
@@ -936,11 +936,26 @@ class Encore {
      *          options.preLoaders = { ... }
      *     });
      *
+     *     // or configure Encore-specific options
```

Comment:
Depending on when it will get merged we'll have to make sure those next lines will be part of the code block added here: https://github.com/symfony/webpack-encore/pull/550/files#diff-168726dbe96b3ce427e7fedce31bb0bcR1000